sdk/cosmos/azure-cosmos/azure/cosmos/partition_key.py

```diff
@@ -109,10 +135,23 @@ def __init__(self, path: List[str], *, kind: Literal["MultiHash"] = "MultiHash",
     def __init__(self, path: str, *, kind: Literal["Hash"] = "Hash", version: int = 2) -> None:
         ...
 
+    @overload
+    def __init__(self, partition_key_definition: Union["PartitionKey", Dict[str, Any]]) -> None:
```

Comment:
We cannot add a new constructor because we are shipping a hotfix. We add an internal helper method if we are refactoring the code.

Reply:
removed constructor and replaced with helper method